### DISCUSSION TITLE
Getting "broken links" on build for a link that exists

### DISCUSSION THREAD

**Discussion by dprothero** [facebook/docusaurus#7246] [Q&A] [answered]
After upgrading from `2.0.0-beta.9` to `2.0.0-beta.18`, I am getting a broken link reported that wasn't previously being reported. Below is a screenshot of the issue.

![image](https://user-images.githubusercontent.com/1897006/165402242-babbd9db-ce8f-4fac-8d7f-619f2b469853.png)

I have tried an absolute URL like `/docs/gateways/rest-api/quickstart/quickstart.md` and get the same issue.

Anyone have any ideas?

**Comment by dprothero** [comment, marked as answer]
I was able to [reproduce this bug](https://github.com/dprothero/docusaurus-broken-links) on a fresh Docusaurus install, so I have opened issue #7247

**Reply by slorber** [reply]
Can't tell why the upgrade broke your site.

Note that your markdown file link (containing a `.md` extension) is clearly not resolved properly, and you end up with the target file not being processed correctly by Docusaurus. You probably have a warning a bit above in the build process to tell you so (related to `onBrokenMarkdownLinks` config that you may have disabled? please avoid)

---


The repro works as intended:

```md
[Test link](/docs/tutorial-basics/quickstart/quickstart.md)
```

=> we never resolve absolute links from site root, as this is not portable with versioning/i18n (copying your `/docs/` folder in another location would break linking)

@Josh-Cena is making this possible in https://github.com/facebook/docusaurus/pull/7248:


```md
[Test link](/tutorial-basics/quickstart/quickstart.md)
```

Link is resolved from `/docs/` folder which is more portable
